eventemitter2.d.ts

```diff
@@ -1,6 +1,7 @@
 declare let eventemitter2: eventemitter2.Static
 
 declare module eventemitter2 {
+    type evetns = string[];
```

Comment:
it's not for events, but for event namespaces in this function IMO. so I think ``eventns`` or ``eventNS`` might be better :)